march_hardware_builder/test/TestAllowedRobots.cpp

```diff
@@ -7,8 +7,8 @@
 #include <march_hardware_builder/HardwareConfigExceptions.h>
 #include <march_hardware_builder/HardwareBuilder.h>
 
-using ::testing::Return;
 using ::testing::AtLeast;
+using ::testing::Return;
```

Comment:
Any reason why you switched this?

Reply:
Automatic formatting, now it's alphabetically sorted